website/pages/docs/plugins/sub-graph.mdx

```diff
@@ -77,7 +77,7 @@ const combinedSchema = builder.toSchema({ subGraph: ['Internal', 'Public'] });
   to being part of the same sub-graph as their parent type. Only applies when type does not have
   `defaultForFields` set.
 
-You can mock any field by adding a mock in the options passed to `builder.buildSchema` under
+You can mock any field by adding a mock in the options passed to `builder.toSchema` under
```

Comment:
I believe you meant to `toSchema` here. Also would you like me to reference the Mocks plugin here, or do you expect this to work out of the box for the SubGraph plugin (I haven't used it yet).

The change would read:

You can mock any field by using the [Mocks plugin](https://pothos-graphql.dev/docs/plugins/mocks) and adding a mock in the options passed to `builder.toSchema` under...

Reply:
I think this was accidentally copied from the mocks plugin, and should just be removed.  The sub-graph plugin doesn't have any special relationship with mocking, and probably shouldn't mention it in the docs.

Reply:
Good, that makes more sense :)